plugins/scaffolder-backend/src/scaffolder/stages/publish/publishers.ts

```diff
@@ -102,12 +102,12 @@ export class Publishers implements PublisherBuilder {
       }
     }
 
-    const gitLabConfig = config.getOptionalConfig('scaffolder.gitlab.api');
+    const gitLabConfig = config.getOptionalConfig('scaffolder.gitlab');
```

Comment:
Is this change just to make it look like the other statements in here for semantics?

I see the change further down too to start using the `api` part too, just wondered the reason for this change

Reply:
Hi @benjdlambert , according to my understanding `scaffolder.gitlab` object is optional. Once user tries to configure(gitlab) object, the `api` attribute/object should be mandatory since  we need mandatory values for `token`. 

Hence, I created the `config.d` file reflecting that and changed the code accordingly.

Reply:
Yep OK - it's a little better validation I guess.